scripts/background.js

```diff
@@ -42,14 +42,25 @@ setInterval(() => {
                     });
                 };
             } else if (!regex.settings.test(url) && !regex.shopping.test(url) && !audio.paused && !audio.src.includes(audioSource.health)) {
-                audio.pause();
+                fadeOutAudio();
             };
         });
 }, 1000);
 
+function fadeOutAudio() {
+    const initialVolume = audio.volume;
+    const fadeOutInterval = setInterval(() => {
+	audio.volume = Math.max(0, audio.volume - 0.01);
```

Comment:
I thibnk this now sounds best.

```suggestion
	audio.volume = Math.max(0, audio.volume - 0.005);
```